docs/source/using-discovery/slurm.md

```diff
@@ -0,0 +1,769 @@
+
+:::{code} bash
+scontrol hold <jobid>
+:::
+
+Release a held job, i.e., permit specified job to start (see hold).
+
+:::{code} bash
+scontrol release <jobid>
+:::
+
+Re-queue a completed, failed, or cancelled job
+
+:::{code} bash
+scontrol requeue <jobid>
+:::
+
+For more information on the commands listed above, along with a complete list of `scontrol` commands run below:
+
+:::{code} bash
+scontrol --help
+:::
+
+### Examples using scontrol
+View information about a specific node:
+
+:::{code} bash
+scontrol show node -d <node_name>
+:::
+
+For information on all reservations, this command will show information about a specific node in the cluster, including the node name, state, number of CPUs, and amount of memory:
+
+:::{code} bash
+scontrol show reservations
+:::
+
+View information about a specific job. This command will show information about a specific job, including the job ID, state, username, and partition name:
+
+:::{code} bash
+scontrol show job <job_id>
+:::
+
+To view information about a specific reservation (e.g., found via `scontrol show res` listed above), and print information about a specific reservation in the cluster, including the reservation name, start time, end time, and nodes included in the reservation:
+
+:::{code} bash
+scontrol show reservation <reservation_name>
+:::
+
+# Advanced Usage
+Advanced usage of Slurm involves working with multi-node jobs, GPU jobs, and understanding priority and Quality of Service (QoS) parameters. It also involves memory management and the use of environment variables in job scripts.
+
+## Multi-node Jobs
+Multi-node jobs involve executing a single job across multiple nodes. Such jobs are typically used for computationally intensive tasks that require significant parallelization.
+
+### Use-cases: Multi-node Jobs
+Multi-node jobs are used in scenarios where tasks can be broken down into sub-tasks that can be executed in parallel, such as simulation and modeling, machine learning training, or big data analysis.
+
+### Code Example for Multi-node Job Submission
+:::{code} bash
+#!/bin/bash
+#SBATCH -J MultiNodeJob     # Job name
+#SBATCH -N 4                # Number of nodes
+#SBATCH -n 16               # Number of tasks
+
+# Your program/command here
+srun ./my_program
+:::
+
+## GPU Jobs
+Slurm can also manage GPU resources, allowing you to specify GPU requirements in your job scripts.
+
+### Use-cases: GPUs
+GPU jobs are used in scenarios where tasks are parallelized and can benefit from the high computational capabilities of GPUs, such as machine learning and deep learning workloads, image processing, or simulations.
+
+### Code Example for GPU Job Submission
+:::{code} bash
+#!/bin/bash
+#SBATCH -J GPUJob            # Job name
+#SBATCH -N 1                 # Number of nodes
+#SBATCH -n 1                 # Number of tasks
+#SBATCH --gres=gpu:2         # Number of GPUs
```

Comment:
this should either be 
#SBATCH -p gpu
#SBATCH --gres=gpu:1

or 
#SBATCH -p multigpu
#SBATCH --gres=gpu:2

Reply:
Good catch!